README.md

```diff
@@ -49,7 +49,39 @@ languages and are part of the [SonarQube](http://www.sonarqube.org/), [SonarClou
 
 ## How to contribute
 
-Check out the [contributing](CONTRIBUTING.md) page to see the best places to log issues and start discussions.
+There are many ways you can contribute to the `sonar-dotnet` project, some very easy and others more involved.
+When contributing, please respect our [Code of Conduct](./CODE_OF_CONDUCT.md).
+
+### Join the discussions
+
+One of the easiest ways to contribute is to participate in discussions on this [GitHub project](https://github.com/SonarSource/sonar-dotnet/issues)
+and on our [Community Forum](https://community.sonarsource.com/).
+
+### Pull Request (PR)
+
+If you want to fix an issue from [the backlog](https://github.com/SonarSource/sonar-dotnet/issues),

```

Comment:
Not sure, but seeing how github is using the term `issues` already, we could maybe drop the `from [the backlog]` part and link the issues page with the `issue` keyword